src/main/java/org/sonarsource/scanner/cli/Conf.java

```diff
@@ -244,8 +244,8 @@ private static Path locatePropertiesFile(@Nullable Path defaultPath, Properties
 
   private static Properties toProperties(Path file) {
     Properties properties = new Properties();
-    try (InputStream in = new FileInputStream(file.toFile())) {
-      properties.load(in);
+    try (InputStreamReader reader = new InputStreamReader(new FileInputStream(file.toFile()), UTF_8)) {
```

Comment:
I think it is ok, but it might be a breaking change. Java properties files used to be encoded as ISO-8859-1. Then Java 9  changed to UTF-8, with the following strategy for backward compatibility:
* Try to load the file as UTF-8
* If an invalid UTF-8 byte sequence is detected, the Java runtime automatically rereads the file in ISO-8859-1.

I think your change is going to remove the second part (to be confirmed, you can try to use some ISO-8859-1 characters that don't map to UTF-8 to reproduce if you have time). I think this is ok to do this move, as UTF-8 is now the standard, and it makes things simpler, but we might have to document it, and maybe change the Scanner CLI major version when releasing.

Reply:
I tried to reproduce, but no matter what I did, the test still passed with my change and failed with the previous version.

Reply:
Okay I finally got one (I was not using the good characters :s)

Reply:
I created version 7.0 in Jira and updated the ticket version with it.
In the meantime, I opened another PR to add the releasabillity workflow (we did not have it on this repository). Hopefully, it means that we will not be able to release a new minor `6.x.y` version, once this is merged.